tests/core/test_expectations.py

```diff
@@ -2062,7 +2062,81 @@ def _error_threshold_exceeds(expectations):
                                      {"row_dq_status": "Passed", "source_agg_dq_status": "Passed",
                                       "final_agg_dq_status": "Passed", "run_status": "Passed",
                                       "source_query_dq_status": "Passed", "final_query_dq_status": "Passed"}  # status
-                             )
+                             ),
+                            (
+                             # Test case 23
```

Comment:
Can you please description about `parameterized query` example in the description, so that it is easily recognized - while we read this later

Reply:
added comments in description